alignak_backend/timeseries.py

```diff
@@ -236,3 +250,30 @@ def send_to_timeseries_influxdb(data, influxdb):
             return True
         except:  # pylint: disable=W0702
             return False
+
+    @staticmethod
+    def send_to_statsd(data, statsd_id):
+        """
+        Send data to statsd
+
+        :param data: list of perfdata to send to statsd
+        :type data: list
+        :param statsd_id: id of statsd
+        :type statsd_id: str
+        :return: True (because statsd not have return error or not)
+        :rtype: bool
+        """
+        statsd_db = current_app.data.driver.db['statsd']
+        item = statsd_db.find_one({'_id': statsd_id})
+        statsd_inst = statsd.StatsClient(item['address'], item['port'])
+        for d in data:
+            if d['service'] == '':
+                prefix = '.'.join([d['realm'], d['host']])
+            else:
+                prefix = '.'.join([d['realm'], d['host'], d['service']])
+
+            if d["uom"] in ['s', 'ms']:
+                statsd_inst.timing('.'.join([prefix, d['name']]), d['value'])
+            else:
+                statsd_inst.gauge('.'.join([prefix, d['name']]), d['value'])
```

Comment:
Something I did not see clearly: if statsd is enabled for a `graphite` the backend sends the data to the StatsD **instead** of carbon, right? StatsD is a frontend for carbon